bin/jib.sh

```diff
@@ -1,197 +0,0 @@
-                echo "Trying mirror ${mirror}"
-                jib_url="${mirror}/${jib_path}"
-                ${getcmd} ${jib_url} > "${installed_jib_script}.gz"
-                if [ -s "${installed_jib_script}.gz" ]; then
-                    echo "Download from mirror successful"
-                    break
-                else
-                    echo "Failed to download ${jib_url}"
-                fi
-            done
-            IFS="${OLD_IFS}"
-        fi
-        if [ ! -s "${installed_jib_script}.gz" ]; then
-            exit 1
-        fi
-    fi
-    # Want to check the filetype using file, to see if we got served a HTML error page.
-    # This is sensitive to the filename containing a specific string, but good enough.
-    file ${installed_jib_script}.gz | grep "gzip compressed data" > /dev/null
-    if [ $? -ne 0 ]; then 
-        echo "Warning: ${installed_jib_script}.gz is not a gzip file."
-        echo "If you are behind a proxy you may need to configure exceptions using no_proxy."
-        echo "The download URL was: ${jib_url}"
-        exit 1
-    fi
-    echo "Extracting JIB bootstrap script"
-    rm -f "${installed_jib_script}"
-    gunzip "${installed_jib_script}.gz"
-    chmod +x "${installed_jib_script}"
-    echo "${data_string}" > "${install_data}"
-}
-
-# Returns a shell-escaped version of the argument given.
-shell_quote() {
-  if [[ -n "$1" ]]; then
-    # Uses only shell-safe characters?  No quoting needed.
-    # '=' is a zsh meta-character, but only in word-initial position.
-    if echo "$1" | grep '^[ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789\.:,%/+=_-]\{1,\}$' > /dev/null \
-        && ! echo "$1" | grep '^=' > /dev/null; then
-      quoted="$1"
-    else
-      if echo "$1" | grep "[\'!]" > /dev/null; then
-        # csh does history expansion within single quotes, but not
-        # when backslash-escaped!
-        local quoted_quote="'\\''" quoted_exclam="'\\!'"
-        word="${1//\'/${quoted_quote}}"
-        word="${1//\!/${quoted_exclam}}"
-      fi
-      quoted="'$1'"
-    fi
-    echo "$quoted"
-  fi
-}
-
-# Main body starts here
-
-setup_url
-
-if [ ! -x "${installed_jib_script}" ]; then
-    install_jib
-elif [ ! -e "${install_data}" ] || [ "${data_string}" != "$(cat "${install_data}")" ]; then
-    echo "Install url changed since last time, reinstalling"
-    install_jib
-fi
-
-# Provide a reasonable default for the --src-dir parameter if run out of tree
-if [ -z "${JIB_SRC_DIR}" ]; then
-    export JIB_SRC_DIR="${mydir}/../"
-fi
-
-
-# Save the original command line
-conf_quoted_arguments=()
-for conf_option; do
-  conf_quoted_arguments=("${conf_quoted_arguments[@]}" "$(shell_quote "$conf_option")")
-done
-export REAL_CONFIGURE_COMMAND_LINE="${conf_quoted_arguments[@]}"
-
-myfulldir="$(cd "${mydir}" > /dev/null && pwd)"
-export REAL_CONFIGURE_COMMAND_EXEC_FULL="$BASH $myfulldir/$myname"
-export REAL_CONFIGURE_COMMAND_EXEC_SHORT="$myname"
-
-${installed_jib_script} "$@"
```

Comment:
Don't we need this?

Reply:
I just now noticed this file was deleted, that's not ok.

Reply:
Sorry about that, this was from an earlier mistake. This should be rectified in the following commit.